### DISCUSSION TITLE
Seeking Guidance on NFT Implementation Using Ignite

### DISCUSSION THREAD

**Discussion by hiroyukikumazawa** [cosmos/community#80] [Chain Builders (Cosmos SDK)] [answered]
Hello Cosmos Community,

Does anyone have experience or a guide on setting up NFTs with Ignite that they could share? Additionally, I would appreciate any tips on potential pitfalls or essential considerations when building this kind of application.

Thank you in advance for your insights and assistance. I look forward to contributing to the community and learning from your experiences.

Best regards,
Hiroyuki

**Comment by julienrbrt** [comment, marked as answer]
Hi,

You can use Cosmos SDK NFT module as a base for your NFT module (e.g https://github.com/OmniFlix/omniflixhub/blob/aa0c729bb4e82bc007207016feb73a2e6f80dc34/x/onft/keeper/onft.go#L60-L87). The documentation can be found here: https://docs.cosmos.network/v0.50/build/modules/nft#nft.

**Reply by hiroyukikumazawa** [reply]
Will try that.
Thank you.
